code/modules/client/preferences_factions.dm

```diff
@@ -13,58 +13,84 @@ var/global/list/seen_religions = list()
 	return
 
 var/global/list/citizenship_choices = list(
-	"Sol Alliance",
 	"Republic of Biesel",
+	"Sol Alliance",
+	"Frontier Alliance",
 	"Republic of Elyra",
 	"Eridani Federation",
+	"Empire of Dominia",
 	"Jargon Federation",
 	"People's Republic of Adhomai",
 	"Izweski Hegemony",
 	"Zo'ra Hive",
 	"K'lax Hive",
-	"Empire of Dominia"
+	"C'thur Hive"
```

Comment:
There is no C'thur citizenship. They're all citizens of Jargon Federation.

Reply:
thanks